src/np_client/counterparty.rs

```diff
@@ -0,0 +1,63 @@
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use self::addresses::GetAddressesBuilder;
+use self::contact_persons::{ContactPerson, GetContactPersonsBuilder};
+use self::counterparties::GetCounterpartiesBuilder;
+use self::create::{CreateCounterpartyBuilder, NoName, NoPhone, NoType, NoRole};
+
+use super::NPClient;
+use super::helper_structs::CounterpartyType;
+use super::res_template::ResponseTemplate;
+
+mod addresses;
+mod contact_persons;
+mod create;
+mod counterparties;
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct Counterparty {
+    r#ref: Uuid,
+    description: String,
+    first_name: String,
+    middle_name: String,
+    last_name: String,
+    counterparty: Option<String>,
+    ownership_form: Option<String>,
+    ownership_form_ref: Option<String>,
+    ownership_form_description: Option<String>,
+    EDRPOU: Option<String>,
```

Comment:
In my opinion, it still better to follow the guidelines and use lowercase variable name. If you will have issues with (de)serializaition, serde provides `#[serde(rename = "name")]` attribute.

Reply:
Indeed! For some reason, I haven't thought about that 🤦‍♀️